### ISSUE TITLE
Improve documentation in using snippets

### ISSUE THREAD

**Issue by versacodes** [vqns/lite-xl-snippets#2]
It would be nice if there is a good tutorial/documentation for creating a snippet. Like what files to create or some template files for easier making of snippets.

**Comment by vqns** [comment]
> Like what files to create

For json files, you need to add them to the lsp snippet plugin; for lua based configuration, it can be anywhere the editor automatically loads from (e.g your `init.lua` or in the plugin path), or you can also manually load the files yourself (`dofile '/path/to/file'`).

> some template files for easier making of snippets

For lsp style, you can refer to the spec or existing snippet collections (see the readme notes). Otherwise, there's a fairly comprehensive doc file (`docs.md`).

I'll add the loading part to the readme, is there anything else you think is missing or not clear?

**Comment by versacodes** [comment]
Fixed it. It should be only the files(snippets, lsp_snippets and json.lua) that should be cloned and put in the plugins folder.

What I did was clone the whole repo which made a _lite-xl-snippets_ folder. But I liked this more since it's organized so I just fixed the required paths in the lsp_snippets.lua file including the _json for loop_ and added _snippets.add_paths_ in init.lua.

Thanks for the help :)
